SQLClient/src/cmd/addCountyAnnotation.go

```diff
@@ -0,0 +1,60 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/ECE356-Final-Project/SQLClient/src/internal/queryMaker"
+	"github.com/spf13/cobra"
+)
+
+const addCountyAnnotation = "AddCountyAnnotation"
+
+var state string
+var countyToAnnotate string
+var annotation string
+
+var addCountyAnnotationCmd = &cobra.Command{
+	Use:   addCountyAnnotation,
+	Short: "Add an annotation string to a county. Can also be used to mark counties of interest",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		qm := queryMaker.NewQueryMaker()
+		defer qm.Db.Close()
+
+		doesCountyExist, attemptedQuery, err := qm.CheckCountyExists(state, countyToAnnotate)
```

Comment:
this is probably fine here but genearlly for testing, we should have all this logic live in one combined function in queryMaker. if we do that and have one Querymaker function for each cmd, we can test most of entire cli logic with just the QM tests